vscode-wpilib/resources/gradle/c/build.gradle

```
@@ -28,6 +28,8 @@ deploy {
                 frcStaticFileDeploy(getArtifactTypeClass('FileTreeArtifact')) {
                     files = project.fileTree('src/main/deploy')
                     directory = '/home/lvuser/deploy'
+                    deleteOldFiles = false // Change to true to delete files on roboRIO that no
+                                           //  onger exist in deploy directory on roboRIO

```

```suggestion
                                           //  longer exist in deploy directory on roboRIO

```

and similar for the other files.